packages/core/src/card/Card.css

```diff
@@ -1,23 +1,25 @@
 /* Styles applied to the root element */
 .saltCard {
+  --card-borderColor: var(--salt-container-primary-borderColor);
+  --card-shadow: var(--salt-overlayable-shadow);
+
   background: var(--saltCard-background, var(--salt-container-primary-background));
   border-width: var(--saltCard-borderWidth, var(--salt-size-border));
   border-style: var(--saltCard-borderStyle, var(--salt-container-borderStyle));
-  border-color: var(--saltCard-borderColor, var(--salt-container-primary-borderColor));
+  border-color: var(--saltCard-borderColor, var(--card-borderColor));
   border-radius: var(--saltCard-borderRadius, 0);
-  box-shadow: var(--saltCard-boxShadow, var(--salt-overlayable-shadow));
-  color: var(--saltCard-color, var(--salt-text-primary-foreground));
+  box-shadow: var(--saltCard-boxShadow, var(--card-shadow));
   overflow: hidden;
-  transition: box-shadow 0.3s cubic-bezier(0.4, 0, 0.2, 1);
+  transition: box-shadow 0.3s cubic-bezier(0.4, 0, 0.2, 1); /* TODO: this should possibly be moved to animation.css? */
 }
 
 /* Styles applied to Card content */
 .saltCard-content {
-  color: inherit;
-  font-size: var(--saltCard-content-fontSize, var(--salt-text-fontSize));
-  letter-spacing: var(--saltCard-content-letterSpacing, normal);
-  line-height: var(--salt-text-lineHeight);
-  min-height: var(--saltCard-content-minHeight, var(--salt-text-minHeight));
+  color: var(--saltCard-color, var(--salt-text-primary-foreground));
+  font-size: var(--saltCard-fontSize, inherit);
```

Comment:
I'm not sure about making text styles configurable inside Card. I don't quite see the point